### ISSUE TITLE
Eyedropper not on by default & harder to use

### ISSUE THREAD

**Issue by aryehsilver** [MicrosoftEdge/DevTools#85]
In a recent update the eyedropper (from the colour picker) has been redesigned. Before this update the eyedropper [was on by default](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/reference#sample-a-color-off-the-page-with-the-eyedropper) (see the first line of the paragraph linked) when opening the colour picker. This enabled selecting a colour from the page without requiring any additional interaction. After the redesign, the eyedropper is no longer on by default.

Secondly before the redesign the colour picker used to be near the cursor. Meaning when selecting a colour from the page you would put the cursor where you want to select and you would get the circle near it. This meant you could actually see what you were selecting.
![image](https://user-images.githubusercontent.com/55258552/180629244-114355f2-3676-4f60-ab7e-9b3023b401de.png)
See the cursor in red, (image adapted from the above link with cursor in red for visibility).
After the redesign though, the cursor has been changed to be the circle where you see the colour being hovered, this makes it harder to select an exact colour.
![image](https://user-images.githubusercontent.com/55258552/180629309-367e5f28-fc6d-467c-9c01-dfada5aefa28.png)
So much harder to pinpoint an exact colour!

**Comment by aryehsilver** [comment]
Annoyingly the second issue seems to be a design choice by the Chromium team:

> We have previously evaluated having an independent cursor on the left side of the magnifying glass[1], but due to having to reposition the magnified view while the cursor is in the corners and for consistency across the desktop OSes (on Mac we are using the NSColorSampler) we have decided to use a center follow approach.
https://bugs.chromium.org/p/chromium/issues/detail?id=992297

See this issue: https://bugs.chromium.org/p/chromium/issues/detail?id=1245191

**Comment by captainbrosset** [comment]
Indeed, this was a change done by the Google team to Chromium and is by design. Here is some discussion about it: https://bugs.chromium.org/p/chromium/issues/detail?id=1225198

You now need to click the eydropper icon to enable it. I personally find this much less confusing than what it was before.
Maybe worth adding a comment on the issue I linked to asking if a setting could be added to switch it to "on by default"?

**Comment by aryehsilver** [comment]
Thank you @captainbrosset for clarifying.
Shame that people got confused by it - it was so natural for me...
Would be useful if there was a setting for the user to decide on the default behaviour.
